packages/apps/spaces/components/organization/organization-list/OrganizationListColumns.tsx

```diff
@@ -107,4 +109,23 @@ export const columns = [
     header: (props) => <THead<Organization> title='Owner' {...props} />,
     skeleton: () => <Skeleton width='100%' height='21px' />,
   }),
+  columnHelper.accessor('lastTouchPointAt', {
+    id: 'LAST_TOUCHPOINT',
+    cell: (props) => (
+      <LastTouchpointTableCell
+        lastTouchPointAt={props.row.original.lastTouchPointAt}
+        lastTouchPointTimelineEvent={
+          (props.row.original as Organization).lastTouchPointTimelineEvent
+        }
+      />
+    ),
+    header: (props) => (
+      <THead<Organization>
+        title='Last touchpoint'
+        subTitle={'How long ago'}
+        {...props}
```

Comment:
THead is missing `columnHasIcon` prop in order to be correctly aligned with the column cells.